tools/keytools/otp/otp-keystore-gen.c

```diff
@@ -0,0 +1,91 @@
+/* otp-keystore-primer.c
```

Comment:
otp-keystore-gen.c

Reply:
This seems like a useful generic tool not specific to OTP that allows a .bin version of the keystore. This would be useful if someone wanted to store the keystore in a specific location or flash in a different way.

Reply:
"keystore.der" is already a binary version of the keystore. This otp.bin also includes the "magic" number at the beginning to be specifically stored in OTP. Other storage might want to use custom headers, as they will have to provide the keystore functions.